src/components/OutsideClickHandler.jsx

```diff
@@ -1,16 +1,16 @@
 import React, { PropTypes } from 'react';
-import { forbidExtraProps } from 'airbnb-prop-types';
+// import { forbidExtraProps } from 'airbnb-prop-types'; // TODO: add to propTypes; semver-major
```

Comment:
That's a tough call; since someone could always have imported it directly by a deep file path, i'd say yes. Some say no, since it's not a documented entry point.